src/Schema.php

```diff
@@ -62,6 +62,22 @@ public static function ensureUploadsHtaccess() {
     static::createOrPrependFile($pathname, $template);
   }
 
+  /**
+   * Ensures fast 404 responses for missing files in uploads folder.
+   */
+  public static function ensureFast404Response() {
+    $uploads_dir = wp_get_upload_dir()['basedir'];
+    $uploads_dir_relative = substr($uploads_dir, strlen(ABSPATH));
+
+    // Changes to .htaccess need to be performed in separate steps for each
+    // chunk of content that needs to be ensured. Otherwise the existing chunks
+    // would be duplicated.
+    $template = file_get_contents(Plugin::getBasePath() . '/conf/.htaccess.uploads.fast404');
+    $template = str_replace('UPLOAD_DIR_REPLACE', $uploads_dir_relative, $template);
+    static::createOrPrependFile(ABSPATH . '.htaccess', $template, "\n");
+    static::removeFromFile($uploads_dir . '/.htaccess', $template);
```

Comment:
`$template` needs to be the original snippet from `conf/.htaccess.uploads.fast404` from before this PR.

I would recommend to copy the original file `conf/.htaccess.uploads.fast404` into a subdirectory `conf/remove/.htaccess.uploads.fast404`